cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -150,54 +167,35 @@ func (s *VMTServer) Run(_ []string) error {
 		select {}
 	}
 
-	if !s.LeaderElection.LeaderElect {
-		glog.Infof("No leader election")
-		run(nil)
-		glog.Fatal("this statement is unreachable")
-		panic("unreachable")
-	}
+	go startHttp(s)
 
-	id, err := os.Hostname()
-	if err != nil {
-		return err
-	}
-
-	leaderelection.RunOrDie(leaderelection.LeaderElectionConfig{
-		EndpointsMeta: api.ObjectMeta{
-			Namespace: "kube-system",
-			Name:      "kubeturbo",
-		},
-		Client:        leaderElectionClient,
-		Identity:      id,
-		EventRecorder: vmtConfig.Recorder,
-		LeaseDuration: s.LeaderElection.LeaseDuration.Duration,
-		RenewDeadline: s.LeaderElection.RenewDeadline.Duration,
-		RetryPeriod:   s.LeaderElection.RetryPeriod.Duration,
-		Callbacks: leaderelection.LeaderCallbacks{
-			OnStartedLeading: run,
-			OnStoppedLeading: func() {
-				glog.Fatalf("lost master")
-			},
-		},
-	})
+	//if !s.LeaderElection.LeaderElect {
+	glog.V(2).Infof("No leader election")
+	run(nil)
 
 	glog.Fatal("this statement is unreachable")
 	panic("unreachable")
 }
 
-// Set up the profiling web interface if profiling is enabled.
-func (s *VMTServer) profiling() {
-	if !s.EnableProfiling {
-		return
-	}
-	glog.V(2).Info("Profiling is enabled.")
+func startHttp(s *VMTServer) {
 	mux := http.NewServeMux()
-	mux.HandleFunc("/debug/pprof/", pprof.Index)
-	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
-	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+
+	//healthz
+	healthz.InstallHandler(mux)
+
+	//debug
+	if s.EnableProfiling {
+		mux.HandleFunc("/debug/pprof/", pprof.Index)
+		mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+		mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+		mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+	}
+
+	//prometheus.metrics
+	mux.Handle("/metrics", prometheus.Handler())
```

Comment:
I think it is mainly for debug. Will move it into debug.
We can expose the metrics such as how many requests have been processed, the latency, and so on.